beacon_node/execution_layer/src/engines.rs

```diff
@@ -273,7 +276,7 @@ impl Engine {
 
                 let mut state = self.state.write().await;
                 state.update(EngineStateInternal::AuthFailed);
-                **state
+                (**state, CapabilitiesCacheAction::None)
```

Comment:
```suggestion
                (**state, CapabilitiesCacheAction::Clear)
```

I would expect to get an auth error in two cases:

1. We are connecting to the EE for the first time and a `Clear` can't hurt.
1. The EE has changed and it would be good to `Clear` and get fresh capabilities.

It's probably the case that in (2) we'd get an `Offline` first, but using `Clear` for auth errors still seems to make sense to me.